src/GafferUI/ViewportGadget.cpp

```diff
@@ -1911,7 +1911,7 @@ void ViewportGadget::SelectionScope::begin( const ViewportGadget *viewportGadget
 	camera->setTransform( viewportGadget->getCameraTransform() );
 	/// \todo It would be better to base this on whether we have a depth buffer or not, but
 	/// we don't have access to that information right now.
-	m_depthSort = camera->isInstanceOf( IECoreGL::PerspectiveCamera::staticTypeId() );
+	m_depthSort = camera->isInstanceOf( IECoreGL::Camera::staticTypeId() );
```

Comment:
This is interesting @danieldresser-ie. In https://github.com/ImageEngine/cortex/commit/cecdf79a5a61f5821a099e355de6f9a4b1cb8ea2 we removed `IECoreGL::PerspectiveCamera`, but forgot to remove the header. GCC and Clang let us get away with it but MSVC has pointed it out.

So I think `m_depthSort` has been unconditionally `false` ever since. I don't recall the selection code that well, but i think perhaps this would only have caused a problem with the legacy `GL_SELECT` mechanism, because the ID-based one will only see the frontmost object anyway. And I think this check was really a hack to determine if we were in the Viewer (where we have a depth buffer and want depth sorting) or the GraphEditor (where we don't).

If that's correct @ericmehl, then this commit will make `m_depthSort` unconditionally `true` instead. We'll need to do some investigation to see if that's problematic or not...